client.js

```diff
@@ -238,7 +238,7 @@ class Player {
 	}
 }
 
-let angle = 0;
+var angle = 0;
```

Comment:
Why are we downgrading our beautiful `let`?

Reply:
Also thought it was causing issues.